src/game.cpp

```diff
@@ -2127,6 +2129,8 @@ bool Game::connectToServer(const std::string &playername,
 	if (!client)
 		return false;
 
+	client->m_simple_singleplayer_mode = this->simple_singleplayer_mode;
```

Comment:
not too happy with this
how about renaming this to `m_want_confirm_register`?

Reply:
No, IMO.

Reply:
Please remove the this